.github/workflows/blc.yml

```diff
@@ -0,0 +1,31 @@
+name: Broken link checker
```

Comment:
Also, can you add `name` fields to steps and comment them similar to https://github.com/oasisprotocol/oasis-core/blob/master/.github/workflows/ci-lint.yml?